drivers/Ide.c

```
@@ -183,8 +184,7 @@ int IdeInit(void) {
                     model[i] = '\0';
                 }
 
-                char dev_name[16];
-                FormatA(dev_name, sizeof(dev_name), "hd%c", 'a' + (channel * 2 + drive));
+                const char* dev_name = GenerateDriveName(DEVICE_TYPE_IDE);
```

Clang (14.0.6)

[warning] variable 'dev_name' is not initialized

(cppcoreguidelines-init-variables)